Scenes/LoginScreen.cs

```diff
@@ -26,21 +28,12 @@ private void _on_JoinButton_button_up()
     _serilogger.Information($"LoginScreen: trying to login as {textField.Text}");
     MyGame.myUuid = textField.Text;
 
-    //EmitSignal("SetPlayerName", textField.Text);
-    bool success = MyGame.JoinGameAsPlayer(textField.Text);
-    if (!success)
-    {
-      _serilogger.Information($"LoginScreen: join failed TODO tell player why");
-      // TODO: alert errors or something 
-    }
-    else
-    {
-      // since we successfully joined the game, we can remove this node
-      // which is the login screen. removing the login screen "displays"
-      // the main game window
-      QueueFree();
-      MyGame.initializeGameUI();
-    }
+    MyGame.goToTheGame(textField.Text);
```

Comment:
This is the case the auth is disabled. Basically I copied the the logic from this module and I created a function to call it in case we use the auth process or not